plugin/plugin.go

```diff
@@ -52,7 +54,7 @@ func init() {
 
 			snapshotter, err := aufs.New(root)
 			if err != nil {
-				return nil, errors.Wrap(plugin.ErrSkipPlugin, err.Error())
+				return nil, fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin)
```

Comment:
accurate port... but it feels it feels like the wrap is backwards maybe it should read:
fmt.Errorf("%s: %w", plugin.ErrSkipPlugin.Error(), err)

Reply:
Apologies, I was wrong you had it right the first time with fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin